creator-node/src/serviceRegistry.js

```diff
@@ -92,6 +95,22 @@ const initAudiusLibs = async () => {
   return audiusLibs
 }
 
+/**
+ * Get all creator nodes registered on chain. Use a cache to avoid making execessive amounts of chain calls
+ * TODO - make this a job that pulls this data more frequently
+ */
+const writeAllRegisteredCNodesToRedis = async (libs) => {
```

Comment:
The reason this is here is because having a libs (from serviceRegistry) dependency in utils or any commonly imported file causes a circular dependency. serviceRegistry -> snapback -> utils -> serviceRegistry. We can remove the dependency on utils from snapback but i didn't want to introduce any other regressions so i didn't want to do that